setup.py

```diff
@@ -91,9 +91,9 @@ def find_version(*file_paths):
         'Driver for allowing Django to use MongoDB as the database backend.'),
     install_requires=[
         'sqlparse==0.2.4',
-        'pymongo>=3.2.0',
+        'pymongo==3.8.0',
         'django>=2.0',
-        'dataclasses>=0.1',
+        'dataclasses==0.6',
```

Comment:
Again why the == requirement

Reply:
Hi @nesdis. This is only fix to tests, I'm changed again